src/bin/nydusd/fs_cache.rs

```diff
@@ -248,6 +250,7 @@ pub struct FsCacheHandler {
     state: Arc<Mutex<FsCacheState>>,
     poller: Mutex<Poll>,
     waker: Arc<Waker>,
+    cache_dir: Mutex<PathBuf>,
```

Comment:
Seems `cache_dir` is only initialized once and never updated, so no need for `Mutex` here?